ONSdigital/census-rm-toolbox
# 2121 Make bad message minimum seen count config, default to 4

# Checklist
Reminder: If any change made to this repo affects the acceptance tests then the version pin in the Pipfile in [census-rm-acceptance-tests]() needs to be updated.

# Motivation and Context
We want to increase the minimum seen threshold on the bad message wizard to exclude more false positives. Also making it configurable.

# What has changed
* Make bad message minimum seen count configurable
* Default bad message minimum seen count to 4

# How to test?
Build, and run the branch image, try the bad message wizard. Should still function exactly the same, just with a higher seen count threshold for bad messages being displayed.

# Links
https://trello.com/c/XYJL16hT/2121-update-bad-message-alerts-8
